sql/functions.sql

```diff
@@ -336,7 +344,7 @@ BEGIN
   IF pzb = 'yes' THEN
     RETURN 2;
   END IF;
-  IF (pzb = 'no' AND lzb = 'no' AND etcs = 'no') OR (atb = 'no' AND etcs = 'no') OR (scmt = 'no' AND etcs = 'no') THEN
+  IF etcs = 'no' AND pzb = 'no' AND lzb = 'no' AND atb = 'no' AND kvb = 'no' AND tvm = 'no' AND scmt = 'no' THEN
```

Comment:
This changes the way this works. The idea is that when we are e.g. in Germany there are only 3 train protection systems that are likely to exist, so when I tag pzb, lzb, and etcs as not exist on a line than there is basically no protection. It doesn't make sense to tag a German line also with tvm=no and all the other tags. And one would have to update all German lines without train protection every time a new system is supported.

Yes, this indeed a complicated way because we now have one key per system and values yes/no/level instead of a key for train protection and values would be a list of supported things. But that is an entirely different discussion.

Reply:
The logic is different from what I understood, but it makes sense. I will fix my commit accordingly!